### PR TITLE
docs: add `examples` dir with create order PII example [MX-1107]

### PR DESCRIPTION
**Pull request by shamrt**
A quick example script for developer users. We'll add more in future.

### CODE REVIEW

File: examples/create-order-with-pii.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+RO.config.set('apiServerUrl', config.apiServerUrl);
+RO.config.set('apiVersion', config.apiVersion);
+RO.config.set('clientId', config.clientId);
+RO.config.set('clientSecret', config.clientSecret);
+
+// PII-specific setup
+RO.config.set('supportedLocales', config.supportedLocales);
+RO.config.set('piiServerUrl', config.piiServerUrl);
+
+const program = RO.program(516, 'national_australian_bank');
```
Comment (Alex-Joseph) [inline examples/create-order-with-pii.js:25]: Question: should we add what the params are?

```suggestion
// RO.program(programId, programCode);
const program = RO.program(516, 'national_australian_bank');
```

File: examples/create-order-with-pii.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+// PII create
+util
```
Comment (Alex-Joseph) [inline examples/create-order-with-pii.js:28]: 👌🏻

**Review by Alex-Joseph — APPROVED** [review]
LGTM, left a suggestion

File: examples/create-order-with-pii.js (hunk 1 above)
Comment (shamrt) [reply examples/create-order-with-pii.js]: Good call.

**Review by Alex-Joseph — APPROVED** [review]
Ah much better! Can't wait for TS!!!

**Pull request merged**